lib/matplotlib/backends/backend_agg.py

```diff
@@ -119,8 +119,15 @@ def _update_methods(self):
         # self.draw_path_collection = self._renderer.draw_path_collection
         self.draw_quad_mesh = self._renderer.draw_quad_mesh
         self.copy_from_bbox = self._renderer.copy_from_bbox
-        self.get_content_extents = self._renderer.get_content_extents
 
+    @cbook.deprecated("3.4")  # Also needs to be removed at C-level.
```

Comment:
It's removed? So don't need this comment?

Reply:
indeed, forgot to update this.